### ISSUE TITLE
"Don't Use Internet Explorer"

### ISSUE THREAD

**Issue by akaplo** [umts/pvta-multiplatform#321]
Might be useful to include a "you should be using a recent version of a real browser" message.  Screenreader functionality seems to be completely missing in IE (it looked like our tester was using either IE 9, 10, or 11 on Windows 7)

**Comment by werebus** [comment]
I just want to reiterate my :-1: There is no way that IE doesn't have screenreader functionality.

**Comment by fermion** [comment]
From a legally blind guy (really) on my team:

<img width="882" alt="screen shot 2017-03-24 at 8 47 31 am" src="https://cloud.githubusercontent.com/assets/2804/24294847/98289164-106e-11e7-8b07-01454ff7b36a.png">

**Comment by akaplo** [comment]
Perhaps, then, it's just that something was wrong with our blind tester's computer?  In general, [Angular supports IE > 8](https://docs.angularjs.org/guide/ie), so there's technically no reason why it wouldn't work.  I'm open to altering the text we merged (sorry @werebus; a thumbs-down with no comment didn't end up on my radar haha).

FWIW, the text we added reads 

>Welcome to pvtrack. 
For optimal screen-reader usage we recommend using NVDA or VoiceOver along with the newest versions of Google Chrome or Safari. 
We have found that using this app on Internet Explorer can cause significant issues with screen readers.

It is the case, in general, that this app (and any app designed since 2015) works best in the newest versions of Chrome, Safari, or Firefox.  Maybe we could alter the text to something more like

> For the best experience, we recommend Chrome with NVDA (Windows) or Safari with VoiceOver (macOS).  pvtrack is not supported on IE 8 or below.

I'll reopen this, since we're discussing it.

**Comment by akaplo** [comment]
@sherson agrees with my comment, and suggested perhaps simply removing the sentence about IE in general, but is not opposed to the rewrite.

**Comment by akaplo** [comment]
New associated PR is #346 (v2 only).

**Comment by akaplo** [comment]
Closed, yet again, via #346.
